src/xcode/ENA/ENA/Source/Scenes/AppInformation/AppInformationViewController+DynamicTableViewModel.swift

```diff
@@ -64,10 +64,16 @@ private extension DynamicCell {
 		}
 	}
 
-	static func html(url: URL?) -> Self {
+	static func html(url: URL?, isInteractive: Bool = false) -> Self {
```

Comment:
Why is there `isInteractive`? Is it ever used with `false` in this context?

Reply:
I wanted to leave the option to have html cells that are non-interactive, in order to be as little as invasive into the existing code base as possible. Is there any reason why this should not be done?

Reply:
Just got confirmed that this cell is used absolutely nowhere else. I will remove the parameter, as all .html cells are supposed to be interactive.